src/Evolution/DiscontinuousGalerkin/Limiters/MinmodHelpers.hpp

```diff
@@ -33,6 +34,16 @@ struct hash;
 namespace Limiters {
 namespace Minmod_detail {
 
+// Encodes the return status of the minmod_tvbm function.
+struct MinmodResult {
+  const double value;
+  const bool activated;
+};
+
+// The TVBM-corrected minmod function, see e.g. Cockburn reference Eq. 2.26.
+MinmodResult minmod_tvbm(double a, double b, double c,
```

Comment:
`minmod_tvbm` ->` tvb_corrected_minmod`

Reply:
@kidder On closer examination, to make this change consistently (in this function, and several variables and documentation) would create conflicts with the next PR I have ready to submit after this one. So it seems overall easier to improve these names in a future PR.